File: solution/342.power-of-four/power-of-four.c
bool isPowerOfFour(int num) {
    if(num == 0)
        return false;
    int a[30];
    int i = 0;
    a[0] = 1;
    for(i = 1;i < 19;i++)
        a[i] = 4 * a[i - 1];
    for(i = 0;i < 19;i++)
        if(num == a[i])
            return true;
    return false;
}
    

File: solution/342.power-of-four/power-of-four.js
/**
 * @param {number} num
 * @return {boolean}
 */
var isPowerOfFour = function(num) {
    let temp = 1;
    for(let i = 0;i < 16;i++){
        if(temp === num){
            return true;
        }else {
            temp = temp * 4;
        }
    }
    return false;
};